features/networking/multicast.feature

```diff
@@ -138,3 +138,135 @@ Feature: testing multicast scenarios
+      | cat | /tmp/proj1pod3.log |
+    Then the step should succeed
+    And the output should match:
+      | <%= cb.proj1pod1ip %>.*joined \(S,G\) = \(\*, 239.255.254.24\), pinging |
+      | <%= cb.proj1pod2ip %>.*joined \(S,G\) = \(\*, 239.255.254.24\), pinging |
+      | <%= cb.proj1pod1ip %>.*multicast, xmt/rcv/%loss = 5/5/0%                |
+      | <%= cb.proj1pod2ip %>.*multicast, xmt/rcv/%loss = 5/5/0%                |
+      
+    # Check multicast group 239.255.254.24 stream in proj2
+    Given I use the "<%= cb.proj2 %>" project
+    # Enable multicast group 239.255.254.24 stream proj2pod1
+    When I run the :exec background client command with:
+      | pod              | <%= cb.proj2pod1 %>                                                                                   |
+      | oc_opts_end      |                                                                                                       |
+      | exec_command     | sh                                                                                                    |
+      | exec_command_arg | -c                                                                                                    |
+      | exec_command     | omping -m 239.255.254.24 -c 5 -T 10 <%= cb.proj2pod1ip %> <%= cb.proj2pod2ip %> <%= cb.proj2pod3ip %> |     
+    Then the step should succeed
+    # Enable multicast group 239.255.254.24 stream proj2pod2
+    When I run the :exec background client command with:
+      | pod              | <%= cb.proj2pod2 %>                                                                                   |
+      | oc_opts_end      |                                                                                                       |
+      | exec_command     | sh                                                                                                    |
+      | exec_command_arg | -c                                                                                                    |
+      | exec_command     | omping -m 239.255.254.24 -c 5 -T 10 <%= cb.proj2pod1ip %> <%= cb.proj2pod2ip %> <%= cb.proj2pod3ip %> |    
+    Then the step should succeed
+    # Enable multicast group 239.255.254.24 stream proj2pod3
+    When I run the :exec background client command with:
+      | pod              | <%= cb.proj2pod3 %> |
+      | oc_opts_end      |                     |
+      | exec_command     | sh                  |
+      | exec_command_arg | -c                  |
+      | exec_command_arg | omping -m 239.255.254.24 -c 5 -T 10 <%= cb.proj2pod1ip %> <%= cb.proj2pod2ip %> <%= cb.proj2pod3ip %> > /tmp/proj2pod3.log |
+    Then the step should succeed
+
+    # Ensure proj2pod3 interface join to the multicast group 239.255.254.24
+    When I execute on the "<%= cb.proj2pod3 %>" pod:
+      | netstat | -ng |
+    Then the step should succeed
+    And the output should match:
+      | eth0\s+1\s+239.255.254.24 |
+    Given 10 seconds have passed
```

Comment:
@weliang1 can we try not to have hard-coded sleep?

Reply:
@pruan-rht Fix to use pulling 10 seconds. Thanks!